packages/mongo-nav/src/helpers/Icons.tsx

```diff
@@ -464,3 +464,98 @@ export function MegaphoneIcon(props: HTMLElementProps<'svg'>) {
     </svg>
   );
 }
+
+export function DocsIcon(props: HTMLElementProps<'svg'>) {
+  return (
+    <svg viewBox="0 0 15 13" width="24px" height="24px" {...props}>
+      <path
+        fill="currentColor"
+        d="M14.7,1.7c0-0.3-0.2-0.6-0.4-0.9c-0.3-0.2-0.6-0.3-0.9-0.3C9.8,0.7,8.2,1.6,7.6,2.3C7,1.6,5.5,0.7,1.9,0.5
+	C1.6,0.5,1.3,0.6,1,0.8C0.7,1.1,0.6,1.4,0.6,1.7l0,7.8c0.1,0.7,0.6,1.1,1.3,1.2c4.7,0.2,5.3,1.8,5.3,1.8c0.1,0.2,0.2,0.3,0.4,0.3
+	c0,0,0.1,0,0.1,0c0.1,0,0.2-0.2,0.3-0.3c0-0.1,0.8-1.6,5.3-1.8c0.7,0,1.2-0.5,1.3-1.2L14.7,1.7z M7.1,11.1c-0.8-0.6-2.4-1.2-5.3-1.3
+	c-0.2,0-0.3-0.1-0.4-0.3l0-7.7c0-0.1,0.1-0.2,0.1-0.2c0.1-0.1,0.2-0.1,0.3-0.1C6.1,1.6,7,2.9,7.1,3.2V11.1z M13.4,9.7
+	c-2.9,0.1-4.5,0.7-5.3,1.3V3.2c0.2-0.3,1.1-1.6,5.3-1.7c0.1,0,0.2,0,0.2,0.1c0.1,0.1,0.1,0.1,0.1,0.2l0,7.7
+	C13.7,9.6,13.6,9.7,13.4,9.7z"
+      />
+      <path
+        fill="currentColor"
+        d="M11,2.8c-0.1,0-0.2,0.1-0.3,0.1c-0.1,0.1-1.9,2-1.5,3.7c0.1,0.7,0.6,1.2,1.4,1.6v0.3c0,0.3,0.2,0.5,0.5,0.5
+	s0.5-0.2,0.5-0.5V8.3c0.8-0.4,1.2-0.9,1.4-1.6C13.2,5,11.4,3.1,11.3,3C11.2,2.9,11.1,2.8,11,2.8z M11.9,6.5c-0.1,0.4-0.4,0.8-0.9,1
+	c-0.5-0.3-0.8-0.6-0.9-1C9.9,5.7,10.5,4.6,11,4C11.5,4.6,12.1,5.7,11.9,6.5z"
+      />
+    </svg>
+  );
+}
+
+export function DevHubIcon(props: HTMLElementProps<'svg'>) {
+  return (
+    <svg
+      width="22px"
+      height="22px"
+      viewBox="0 0 22 22"
+      version="1.1"
+      {...props}
+    >
+      <g
+        id="Annotations"
+        stroke="none"
+        strokeWidth="1"
+        fill="none"
+        fillRule="evenodd"
+      >
+        <g
+          id="03"
+          transform="translate(-1607.000000, -498.000000)"
+          stroke="currentColor"
+        >
+          <g id="Group-2" transform="translate(1586.000000, 177.000000)">
+            <g id="Nav" transform="translate(0.000000, 192.000000)">
```

Comment:
can we remove all the "id" properties? they might conflict with other ones in the page

Reply:
👍

Reply:
👍